src/pages/store/index.vue

```diff
@@ -951,6 +959,14 @@ export default {
       },
     },
 
+    localizedBookstoreCMSTags() {
+      const isChinese = this.$i18n.locale === 'zh-Hant';
+      return this.bookstoreCMSTags.map(tag => ({
+        ...tag,
+        name: isChinese ? tag.nameZh : tag.nameEn,
+        description: isChinese ? tag.descriptionZh : tag.descriptionEn,
```

Comment:
not sure if destruct and reassign looks cleaner
i.e. `const { nameZh, nameEn, descriptionZh, descriptionEn, ...otherInfo } = tag; return { ...otherInfo, etc... }; `